### ISSUE TITLE
Crash report 3.9-rc-1: RuntimeException in FetchBlogListAbstract

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#2459]
Probably a side effect of the 2 step auth merge.

```
java.lang.ExceptionInInitializerError
       at org.wordpress.android.ui.accounts.SignInFragment.refreshBlogContent(SignInFragment.java:844)
       at org.wordpress.android.ui.accounts.SignInFragment.refreshFirstBlogContent(SignInFragment.java:857)
       at org.wordpress.android.ui.accounts.SignInFragment.access$1900(SignInFragment.java:70)
       at org.wordpress.android.ui.accounts.SignInFragment$11.onSuccess(SignInFragment.java:434)
       at org.wordpress.android.ui.accounts.helpers.FetchBlogListWPOrg.getBlogList(FetchBlogListWPOrg.java:87)
       at org.wordpress.android.ui.accounts.helpers.FetchBlogListWPOrg.fetchBlogList(FetchBlogListWPOrg.java:127)
       at org.wordpress.android.ui.accounts.helpers.FetchBlogListAbstract$1.run(FetchBlogListAbstract.java:27)
Caused by: java.lang.RuntimeException: Can't create handler inside thread that has not called Looper.prepare()
       at android.os.Handler.(Handler.java:121)
       at android.os.AsyncTask$InternalHandler.(AsyncTask.java:607)
       at android.os.AsyncTask$InternalHandler.(AsyncTask.java:607)
       at android.os.AsyncTask.(AsyncTask.java:190)
       at org.wordpress.android.ui.accounts.SignInFragment.refreshBlogContent(SignInFragment.java:844)
       at org.wordpress.android.ui.accounts.SignInFragment.refreshFirstBlogContent(SignInFragment.java:857)
       at org.wordpress.android.ui.accounts.SignInFragment.access$1900(SignInFragment.java:70)
       at org.wordpress.android.ui.accounts.SignInFragment$11.onSuccess(SignInFragment.java:434)
       at org.wordpress.android.ui.accounts.helpers.FetchBlogListWPOrg.getBlogList(FetchBlogListWPOrg.java:87)
       at org.wordpress.android.ui.accounts.helpers.FetchBlogListWPOrg.fetchBlogList(FetchBlogListWPOrg.java:127)
       at org.wordpress.android.ui.accounts.helpers.FetchBlogListAbstract$1.run(FetchBlogListAbstract.java:27)
```

**Comment by kwonye** [comment]
Are there any repro steps for this?

**Comment by maxme** [comment]
Reopening, it's actually an issue on Android 4.0.x

**Comment by maxme** [comment]
> Reopening, it's actually an issue on Android 4.0.x

I reopened because of issues reported in 4.0-rc-1 but the fix (#2557) was merged after 4.0-rc-1 release :facepalm:
